docs/install.md

```diff
@@ -1,93 +1,58 @@
 <!--
 ---
-linkTitle: "Installation"
+linkTitle: "Setting Up Tekton Triggers"
 weight: 2
 ---
 -->
-# Installing Tekton Triggers
+# Setting Up Tekton Triggers
 
-Use this page to add the component to an existing Kubernetes cluster.
+Follow the steps below to set up an official release of Tekton Triggers on your Kubernetes cluster. 
+If you want to test Triggers from `HEAD`, see the
+[Tekton Developer Guide](https://github.com/tektoncd/triggers/blob/master/DEVELOPMENT.md#install-triggers).
 
-- [Pre-requisites](#pre-requisites)
-- [Versions](#versions)
-- [Installing Tekton Triggers](#installing-tekton-triggers-1)
+## Prerequisites
 
-## Pre-requisites
+You must satisfy the following prerequisties to set up Tekton Triggers:
 
-1.  A Kubernetes cluster version 1.18 or later
+* You must have Tekton Pipelines installed on your Kubernetes cluster. For instructions, see [Installing Tekton Pipelines](https://github.com/tektoncd/pipeline/blob/master/docs/install.md).
 
-    If you don't already have a cluster, you can create one for testing with
-    `kind`.  [Install
-    `kind`](https://kind.sigs.k8s.io/docs/user/quick-start/#installation) and
-    create a cluster by running [`kind create
-    cluster`](https://kind.sigs.k8s.io/docs/user/quick-start/#creating-a-cluster).
-    This will create a cluster running locally, with RBAC enabled.
+* Your Kubernetes cluster must be running Kubernetes version 1.18 or above.
```

Comment:
I'd mention this before the point about installing Tekton Pipelines. And I'd keep the bit about being able to create a cluster easily using Kind. It does make things a bit more verbose but IMO makes the docs more approachable for someone reading this for the first time.

Reply:
Restored the `kind` instructions as per review comments and moved the Pipelines requirement after the Kubernetes cluster requirement.